wolfssl/wolfcrypt/settings.h

```diff
@@ -210,6 +210,7 @@
     #define TFM_TIMING_RESISTANT
     #define NEED_AES_TABLES
     #define WOLFSSL_HAVE_MIN
+    #define WOLFSSL_HAVE_MAX
```

Comment:
This is inside the MICROCHIP_PIC32 define.  David has also changed this in his PR here:
https://github.com/wolfSSL/wolfssl/pull/357